themes/bootstrap3/templates/search/facet-list-content.phtml

```diff
@@ -0,0 +1,21 @@
+<ul class="full-facet-list facet__list facet-group" id="facet-list-<?=$this->escapeHtmlAttr($key) ?>">
+  <?php if ($this->page > 1 && !$this->inLightbox && !$this->ajaxLightbox): ?>
+    <a href="<?=$this->escapeHtmlAttr($urlBase . '&facetpage=' . ($this->page - 1) . '&facetsort=' . urlencode($this->sort) . '&contains=' . urlencode($this->contains))?>" class="facet js-facet-prev-page" data-page="<?=($this->page + 1) ?>" data-sort="<?=$this->sort ?>" data-limit="<?=count($this->data) ?>"><?=$this->translate('prev_ellipsis') ?></a>
```

Comment:
data-sort needs to be escaped, just in case. prev_ellipsis ellipsis needs transEsc.